src/mixins/portfolio.js

```diff
@@ -210,15 +219,18 @@ export default {
         await fetchPromise;
         this.isLoading = false;
       }
-      if (!this.sortedCreatedClassIds.length) {
-        this.currentTab = 'collected';
-      }
     },
-    goCollected() {
-      this.currentTab = 'collected';
+    changeTab(tab) {
+      this.$router.push({
```

Comment:
not sure if `replace` should be used here or keep `push`

Reply:
Seem using replace keep the current UX, will chang this